utils/index.d.ts

```diff
@@ -1564,6 +1564,7 @@ export declare namespace AzExtFsExtra {
     export function ensureFile(resource: Uri | string): Promise<void>;
     export function readFile(resource: Uri | string): Promise<string>;
     export function writeFile(resource: Uri | string, contents: string): Promise<void>;
+    export function appendFile(resource: Uri | string, contents: string): Promise<void>;
```

Comment:
Since we're making another PR, can we revisit https://github.com/microsoft/vscode-azuretools/pull/1525#discussion_r1268085326 ?

Reply:
From the other comment thread, I agree that an optional separator parameter with default `\r\n\r\n` is suitable.